### PR TITLE
Update local deployment instructions for Kubeflow

### PR DESCRIPTION
**Pull request by akchinSTC**
Previous instructions included deployment of the entire
Kubeflow stack which ate up system resources. These instructions
will only deploy components necessary to run Kubeflow Pipelines.

Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: docs/source/recipes/deploying-kubeflow-locally-for-dev.md [hunk 1]
```diff
@@ -59,44 +56,33 @@ In this example, we will be performing the steps on a MacOS system
...
+From the root directory of Elyra:
 ```bash
-mkdir -p ${KF_DIR}
-cd ${KF_DIR}
-kfctl apply -V -f ${CONFIG_URI}
+export PIPELINE_VERSION=0.5.1
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
+kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+kubectl apply -k "etc/kubernetes/kubeflow/"
 ```
```
Comment (lresende) [inline docs/source/recipes/deploying-kubeflow-locally-for-dev.md:69]: considering not all users will have elyra cloned locally, should we make this a url?

File: docs/source/recipes/deploying-kubeflow-locally-for-dev.md [hunk 2]
```diff
@@ -59,44 +56,33 @@ In this example, we will be performing the steps on a MacOS system
...
-```
 - Deploy Kubeflow  
+
+From the root directory of Elyra:
 ```bash
-mkdir -p ${KF_DIR}
-cd ${KF_DIR}
-kfctl apply -V -f ${CONFIG_URI}
+export PIPELINE_VERSION=0.5.1
+kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/cluster-scoped-resources?ref=$PIPELINE_VERSION"
```
Comment (lresende) [inline docs/source/recipes/deploying-kubeflow-locally-for-dev.md:66]: does this work or you have to use raw?

File: etc/kubernetes/kubeflow/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - https://github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic?ref=0.5.1
+
```
Comment (lresende) [inline etc/kubernetes/kubeflow/kustomization.yaml:22]: does this work or you have to use raw?

File: docs/source/recipes/deploying-kubeflow-locally-for-dev.md (hunk 2 above)
Comment (akchinSTC) [reply docs/source/recipes/deploying-kubeflow-locally-for-dev.md:66]: this works
`-k, --kustomize='': Process a kustomization directory. This flag can't be used together with -f or -R.`

File: etc/kubernetes/kubeflow/kustomization.yaml (hunk 3 above)
Comment (akchinSTC) [reply etc/kubernetes/kubeflow/kustomization.yaml:22]: this works
`-k, --kustomize='': Process a kustomization directory. This flag can't be used together with -f or -R.`

File: docs/source/recipes/deploying-kubeflow-locally-for-dev.md (hunk 1 above)
Comment (akchinSTC) [reply docs/source/recipes/deploying-kubeflow-locally-for-dev.md:69]: replaced with URL

**Comment by akchinSTC** [comment]
NOTE: URL in readme is pointing to the PR for testing purposes and must be updated to 
`github.com/elyra-ai/elyra/etc/kubernetes/kubeflow-pipelines?ref=master` prior to merge.

**Pull request merged**